### PR TITLE
Adding new eac instructions

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2691

### CODE REVIEW

**Comment by get-itips** [comment]
@dariomws Dario please help me reviewing thank you

File: Exchange/ExchangeOnline/mailbox-migration/migrating-imap-mailboxes/migrate-other-types-of-imap-mailboxes.md [hunk 1]
```diff
@@ -293,12 +295,56 @@ You use a migration batch to migrate groups of email to Microsoft 365 or Office
...
+
+To migrate email successfully, Microsoft 365 or Office 365 needs to connect and communicate with the source email system. To do this, Microsoft 365 or Office 365 uses a migration endpoint. This is a technical term that describes the settings that are used to create the connection. You create the migration endpoint and migration batch in this task.
+
+1. Open the [new Exchange admin center](../../exchange-admin-center.md).
+
+2. Go to **Migration**.
+
+3. Click **Add migration batch**.
+
+4. On the **Add migration batch page**, type the name (no spaces or special characters) of the migration batch in the **Give migration batch a unique name** field, for example, Test5-migration
```
Comment (dariomws) [inline Exchange/ExchangeOnline/mailbox-migration/migrating-imap-mailboxes/migrate-other-types-of-imap-mailboxes.md:314]:
```suggestion
4. On the **Add migration batch page**, type the name (no spaces or special characters) of the migration batch in the **Give migration batch a unique name** field, for example, Test5-migration.
```

**Review by dariomws — COMMENTED** [review]
Look good but I'm wondering if there is a better way to organize it. Because when you read the article, you see steps 3, 4, then step 3 again... It could be not very clear.

**Comment by get-itips** [comment]
> Look good but I'm wondering if there is a better way to organize it. Because when you read the article, you see steps 3, 4, then step 3 again... It could be not very clear.

Yeah I agree, however, steps 1 & 2 are common to both EACs (not EAC related actually).

**Comment by dariomws** [comment]
Maybe including the new EAC steps on step 4 and creating two subtitles:
- Using the legacy EAC
- Using the new EAC

Then in step 3, you can add a note to let the reader know that this step is included in step 4 for the new EAC.

What I say is that the following order is strange for me:
- Step 1
- Step 2
- Step 3
- Step 4
- Step 3
- Step 5
- Step 6

**Comment by dariomws** [comment]
Maybe the following order could be better:
- Step 1
- Step 2
- Step 3
  Note the let the reader know it's only for classic EAC. For new EAC it's included in step 4
- Step 4
   Using classic EAC
   Using new EAC
- Step 5
- Step 6
     Using classic EAC
   Using new EAC

**Review by dariomws — APPROVED** [review]

**Pull request merged**